gyucheol/4일차/완주하지_못한_선수.js

```diff
@@ -0,0 +1,9 @@
+function solution(participant, completion) {
+    participant.sort();
+    completion.sort();
+    for(let i = 0; i < participant.length; i++){
+        if(participant[i] !== completion[i]){
+        return participant[i];
+        }
+    }    
+}
```

Comment:
잘풀었습니다~
컨벤션 지키는 것도 코딩에서 중요해요 ㅎㅎ
```suggestion
function solution(participant, completion) {
    participant.sort();
    completion.sort();
    
    for (let i = 0; i < participant.length; i++) {
        if (participant[i] !== completion[i]) {
            return participant[i];
        }
    }    
}
```